src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -266,6 +266,59 @@ public boolean canBeUsedWith(ColumnsExpression.Kind kind)
             return kind != ColumnsExpression.Kind.MAP_ELEMENT;
         }
     },
+    BETWEEN(16)
```

Comment:
I am wondering why `BETWEEN(16)` is not after 15?

Reply:
> I am wondering why `BETWEEN(16)` is not after 15?

It seemed that the convention in Operator.java was not super strict about ordering operations numerically, and so I opted to put it with the slices because that seemed like a fitting way of ordering in lieu of by index.

Reply:
I was wondering, maybe we want to add a defensive unit test that will check the numbers used and break if someone does something wrong? 

 ```
@Test
    public void testOperatorOrder() {
        int[] expectedValues = {0, 4, 3, 1, 2, 7, 5, 6, 8, 9, 10, 11, 12, 13, 14, 15};
        Operator[] operators = Operator.values();

        assertEquals("Number of operators has changed", expectedValues.length, operators.length);

        for (int i = 0; i < operators.length; i++) {
            assertEquals("Operator " + operators[i].name() + " has incorrect value", expectedValues[i], operators[i].getValue());
        }
    }
```
This test will fail if the number of operators or the b value of any operator is changed. It will also fail if a new operator is added but does not take the next number in line.